docs/release-notes/release-notes-0.16.0.md

```diff
@@ -222,6 +225,7 @@ to refactor the itest for code health and maintenance.
 # Contributors (Alphabetical Order)
 
 * Carla Kirk-Cohen
+* Conner Babinchak
```

Comment:
I think you can go ahead and add yourself as will given you contributed some of the fixups.

Reply:
I'm actually already in this list.